src/App.js

```diff
@@ -59,6 +61,9 @@ function App() {
       setTasks([...tasksNew]);
     }
   };
+  const editTokenHandler = (rawToken) => {
```

Comment:
```suggestion
  const updateTokenHandler = (rawToken) => {
```
I think `updateTokenHandler` is more relevant than `edit`, idea originated from Laravel where we use edit terminology for displaying edit form and update for submitting the edit form request.